CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * Bring back some proto messages that were deleted but still needed for historical queries [#1554](https://github.com/provenance-io/provenance/issues/1554).
 * Fix the `MsgModifyNameRequest` endpoint to properly clean up old index data [PR 1565](https://github.com/provenance-io/provenance/pull/1565).
+* Add `NewUpdateAccountAttributeExpirationCmd` to the CLI [#1592](https://github.com/provenance-io/provenance/issues/1592).
```

Comment:
Once #1595 is merged, this line will need to be moved to the unreleased section.